catalog/crd-managing-resources-from-crds.adoc

```diff
@@ -0,0 +1,18 @@
+[id='crd-managing-resources-from-crds']
+= Managing resources from Custom Resource Definitions
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:context: crd-managing-resources-from-crds
+
+toc::[]
+
+include::modules/crd-custom-resource-definitions.adoc[leveloffset=+1]
+include::modules/crd-creating-custom-resources-from-web-console.adoc[leveloffset=+1]
```

Comment:
Same comment about nesting the modules here. Since the stub intro would be used in two assemblies, I'd try to make it a module, if possible. You might also add comments to the three modules to say that they might need to be used together.